docs/get-started/sumo-logic-ui.md

```diff
@@ -1,115 +1,135 @@
 
-Log Search, Metrics, and Live Tail tabs have additional options you can use to customize your environment. The tabs even stay open when you sign out and sign back in again, so you can start where you left off. Clicking any of the following icons opens a tabbed window.
+Sumo Logic Administrators (Admins) are responsible for managing data collection, data settings, and alerts for their organization. You must have Sumo Logic Admin role privileges to perform these tasks.
 
-![WTS_UI_LogSearch-Metrics-LiveTail.png](/img/get-started/ui/WTS_UI_LogSearch-Metrics-LiveTail.png)
+To manage data in Sumo Logic, do the following:
 
-To access additional Log Search, Metrics, and Live Tail options, do the following:
+1. From the top-right navigation bar, click **Configuration**.<br/><img src={useBaseUrl('img/get-started/config.png')} alt="img/get-started/config.png" width="400"/>
+1. Choose from the following, as needed:
+    * **Collection**. Manage [Collectors and Sources](/docs/send-data/collection), [OpenTelemetry Collection](/docs/send-data/opentelemetry-collector), [Data Collection Status](/docs/manage/ingestion-volume/collection-status-page), [Ingest Budget](/docs/manage/ingestion-volume/ingest-budgets), Archive, and [Data Archiving](/docs/manage/data-archiving).
+    * **Logs**. Manage [fields](/docs/manage/fields), [field extraction rules](/docs/manage/field-extractions), [parsers](/docs/search/search-query-language/parse-operators), [partitions](/docs/manage/partitions-data-tiers), [scheduled views](/docs/manage/scheduled-views), and [data forwarding](/docs/manage/data-forwarding).
+    * **Metrics**. Manage [metrics rules](/docs/metrics/metric-rules-editor), [logs-to-metrics](../metrics/logs-to-metrics.md), and [metrics transformation rules](../metrics/metrics-transformation-rules.md).
+    * **Cloud SIEM Integrations**. Manage [Ingest Mappings](/docs/cse/ingestion/sumo-logic-ingest-mapping), [Log Mappings](/docs/cse/schema/create-structured-log-mapping), [Context Actions](/docs/cse/administration/create-cse-context-actions), [Actions](/docs/cse/administration/create-cse-actions), [Enrichment](/docs/cse/integrations/enrichments-and-indicators), [Automation](/docs/cse/automation).
+    * **Cloud SIEM Entities**. Manage [Groups](/docs/cse/records-signals-entities-insights/create-an-entity-group), [Normalization](/docs/cse/schema/username-and-hostname-normalization), [Custom Types](/docs/cse/records-signals-entities-insights/create-custom-entity-type), and [Criticality](/docs/cse/records-signals-entities-insights/entity-criticality).
+    * **Cloud SIEM Workflow**. Manage [Insight Detection](/docs/cse/records-signals-entities-insights/set-insight-generation-window-threshold), [Insight Statuses](/docs/cse/administration/manage-custom-insight-statuses), [Insight Resolutions](/docs/cse/administration/manage-custom-insight-resolutions), and [Tag Schemas](/docs/cse/administration/create-a-custom-tag-schema).
```

Comment:
Hi @jpipkin1 , when you have the bandwidth, would you be able to review the above docs links to make sure they accurately reflect those topics? I did it from scratch and not 100% sure if it's right.

Reply:
Beautiful work, @kimsauce! I only found a few things to update.